src/CVNBot/ProjectList.cs

```diff
@@ -105,7 +109,28 @@ public void AddNewProject(string projectName, string interwiki)
             Project prj = new Project();
             prj.projectName = projectName;
             prj.interwikiLink = interwiki;
-            prj.rooturl = "https://" + projectName + ".org/";
+            switch(projectName){
+                case "mediawiki.wikipedia":
```

Comment:
Very nice to support Wikivoyage and these custom mappings. I think it's okay to keep in the same pull request, but perhaps as a separate commit. Order can be either way, whichever you prefer.

Reply:
Thanks for review. I'll split the patch :)